javascripts/controllers/DetailCtrl.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+app.controller("DetailCtrl", function($location, $rootScope, $scope, ContactService){
+	// $scope.controller = "Hello, DetailCtrl. Route:/contacts/detail/:id";
```

Comment:
no commented code - put it in or delete it

Reply:
Deleted commented code.